Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.8 as base
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+ENV PATH="${PATH}:/root/.poetry/bin"
+WORKDIR /app
+COPY pyproject.toml poetry.lock /app/
```

Comment:
You've set the WORKDIR, so you can use `COPY my_files ./` to copy stuff into the WORKDIR without repeating "/app".